kustomize/traffic/clusterScoped/kustomization.yaml

```diff
@@ -1,5 +1,25 @@
 bases:
-- ../base
+- ../namespaceScoped
 
-resources:
-- clusterroles.yaml
\ No newline at end of file
+patches:
+- patch: |-
+    - op: replace
+      path: /kind
+      value: ClusterRole
+  target:
+    kind: Role
+  
+# Order matters
+# /roleRef/kind patch should happen before /kind patch
+- patch: |-
+    - op: replace
+      path: /roleRef/kind
+      value: ClusterRole
+  target:
+    kind: RoleBinding
+- patch: |-
+    - op: replace
+      path: /kind
+      value: ClusterRoleBinding
+  target:
+    kind: RoleBinding
```

Comment:
I had some trouble patching the `kind`. Seems like certain methods of patching do no like the `kind` being changed. This method seems to work, however.